src/string/memmem.c

```diff
@@ -34,11 +42,14 @@ static char *fourbyte_memmem(const unsigned char *h, size_t k, const unsigned ch
 #define BITOP(a,b,op) \
  ((a)[(size_t)(b)/(8*sizeof *(a))] op (size_t)1<<((size_t)(b)%(8*sizeof *(a))))
 
-static char *twoway_memmem(const unsigned char *h, const unsigned char *z, const unsigned char *n, size_t l)
+_Checked static _Array_ptr<char> twoway_memmem(_Array_ptr<const unsigned char> h : count(z - h),
```

Comment:
The memmem function call this function with these arguments 
twoway_memmem(h, h+k, n, l). 
and the function signature is 
static char *twoway_memmem(const unsigned char *h, const unsigned char *z, const unsigned char *n, size_t l)

So because the bound for h is count(k) and we do not pass k into this function I used z-h which is h+k -h,